### ISSUE TITLE
Heatwave doesn't convert xml namespaces of xslt files used in HeatDirectory

### ISSUE THREAD

**Issue by FireGiantHelp** [firegiant/HeatWaveSupport#5]
### HeatWave Version

0.9.0.5

### Visual Studio Version

17.3.6

### Repro Steps

Heatwave doesn't convert the xml namespaces of the transform file used in HeatDirectory

The .wixproj have this segment at the end:

```
<Target Name="HarvestFiles" AfterTargets="BeforeBuild">
  <HeatDirectory OutputFile="_MainFiles.wxs" DirectoryRefId="INSTALLDIR" 
                 ComponentGroupName="MainFiles" SuppressCom="true"
                 Directory="$(MainFilesPath)" RunAsSeparateProcess="True"
                 NoLogo="true" SuppressFragments="false" SuppressRegistry="true"
                 SuppressRootDirectory="true" AutoGenerateGuids="true" GenerateGuidsNow="false"
                 ToolPath="$(WixToolPath)" Transforms="HarvestMainFiles.xsl" />
</Target>
```

The HarvestMainFiles.xsl file starts with a list of Wix 3 namespaces:

```
<xsl:stylesheet
  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
  xmlns:wix="http://schemas.microsoft.com/wix/2006/wi"
  xmlns="http://schemas.microsoft.com/wix/2006/wi"
  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension"
  version="1.0" 
  exclude-result-prefixes="xsl wix util">
```

See also: https://github.com/wixtoolset/issues/issues/7019

### Actual Result

The XSLT namespaces are not changed by Heatwave.

### Expected Result

Heatwave should change them to their V4 counterparts:

```
<xsl:stylesheet
  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
  xmlns:wix="http://wixtoolset.org/schemas/v4/wxs"
  xmlns="http://wixtoolset.org/schemas/v4/wxs"
  xmlns:util="http://wixtoolset.org/schemas/v4/wxs/util"
  version="1.0" 
  exclude-result-prefixes="xsl wix util">
```

**Comment by FireGiantHelp** [comment]
We can reproduce this internally. This issue is on our backlog to fix.

**Comment by FireGiantHelp** [comment]
This should be fixed in HeatWave v0.9.2. The XSLT files referenced by Heat tasks should have namespaces fixed up now.

**Comment by tbryn** [comment]
I ran into this bug using HeatWave 1.0.1: namespaces in a referenced XSLT file were not upgraded.
